src/Rings/MPolyMap/flattenings.jl

```diff
@@ -493,6 +493,23 @@ end
   return hom(codomain(flat_S), codomain(flat_T), imgs, check=false)
 end
 
+### Computation of induced morphisms on flattened towers of polynomial rings
+@attr Any function flatten(
+    f::MPolyAnyMap{QQMPolyRing, 
```

Comment:
```suggestion
    f::MPolyAnyMap{MPolyRing,
```
why not make this available for all coefficient rings, instead of only for QQ?

Reply:
yup yup, this was just a cherry from what I needed to get my computations running. Thanks